photon-ml/src/main/scala/com/linkedin/photon/ml/algorithm/RandomEffectCoordinate.scala

```diff
@@ -144,7 +144,9 @@ object RandomEffectCoordinate {
       .join(randomEffectModel.modelsRDD)
       .flatMap { case (randomEffectId, (localDataSet, model)) =>
         localDataSet.dataPoints.map { case (uniqueId, labeledPoint) =>
-          (uniqueId, model.computeScore(labeledPoint.features))
+          (uniqueId,
+            ScoredGameDatum(labeledPoint.label, labeledPoint.offset, labeledPoint.weight,
+              model.computeScore(labeledPoint.features), Map()))
```

Comment:
This can also be alleviated by having a `ScoreGameDatum.apply(LabeledPoint, score...)`. It will make the line shorter.